mlrun/config.py

```diff
@@ -134,6 +134,9 @@
         "resources": {
             "delete_crd_resources_timeout": "5 minutes",
         },
+        "alert_activations": {
+            "retention_days": 14 * 7,
+        },
```

Comment:
While we're dealing only with alert activations at this point, I'm seeing the partitioning mechanism as something we'd like to generalize later to additional tables. So, I'm thinking it's better to reverse the order here, i.e. something like:
```
"object_retentions": {
    "alert_activations": 14*7,
},
```
Then in the `validate` function, you can go over all values in the `object_retentions` section and perform the same validation for all of them.